autogen/agentchat/contrib/graph_rag/falkor_graph_query_engine.py

```diff
@@ -0,0 +1,63 @@
+import os
+from dataclasses import field
+from typing import List
+
+from graphrag_sdk import KnowledgeGraph, Source
+from graphrag_sdk.schema import Schema
+
+from .document import Document
+from .graph_query_engine import GraphStoreQueryResult
+
+
+class FalkorGraphQueryResult(GraphStoreQueryResult):
+    messages: list = field(default_factory=list)
+
+
+class FalkorGraphQueryEngine:
+    """
+    This is a wrapper for Falkor DB KnowledgeGraph.
+    """
+
+    def __init__(
+        self,
+        name: str,
+        host: str = "127.0.0.1",
+        port: int = 6379,
+        username: str | None = None,
+        password: str | None = None,
+        model: str = "gpt-4-1106-preview",
```

Comment:
Do we want to remove the default value and require a user to specify the model name?
Also, since this is a user-facing API, shall we add docstr? Similar for other user-facing APIs in this file.

Reply:
yes, sounds good.

Reply:
Updated the docstr